### DISCUSSION TITLE
.NET CORE

### DISCUSSION THREAD

**Discussion by ferrarinim** [abuzuhri/Amazon-SP-API-CSharp#394] [Q&A] [answered]
hi, I am migrating my project to blazor (.NET Core),
right now i'm using your fantastic library (grateful to you for developing it) but i don't think it will be compatible with the new version of the framework.
Are you planning to update the library and make it compatible with .net core?
Thanks in advance

**Comment by abuzuhri** [comment, marked as answer]
Hi @ferrarinim ,

Library build on netstandard2.0 , so that mean its compatible with (.NET CORE)

**Reply by ferrarinim** [reply]
This is a fantastic news. I could inquire before asking, sorry.

**Comment by abuzuhri** [comment]
Check this documents 

https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0

**Comment by TechyChap** [comment]
Happily been using with .Net Core and now .Net 6.0
